### ISSUE TITLE
Add configurable enabled/disabled formatting

### ISSUE THREAD

**Issue by SlimeDog** [SlimeDog/PluginVersions#14]
By default, use the same formatting/coloration as the native `plugins` command and PluginManager `pm list` command. So three format/color strings.

![Screen Shot 2023-03-17 at 7 14 12 AM](https://user-images.githubusercontent.com/17748923/225929915-ce2f6095-fc28-4eb9-bc37-a469d94c17fe.png)

Make the formatting  strings configurable in `config.yml`.

**Comment by SlimeDog** [comment]
Configuration options:
[Legacy MC](https://minecraft.fandom.com/wiki/Formatting_codes)
```
format-enabled: '&a'
format-disabled: '&c'
format-version: '&e'
```
[MiniMessage](https://docs.advntr.dev/minimessage/format.html), which includes hex formatting
```
format-enabled: '<green>'
format-disabled: '<#f44336>'
format-version: '<yellow>'
```

**Comment by SlimeDog** [comment]
I prefer MiniMessage. And I prefer separate specification of `version`, so that the output can be tabular at the console, as it was prior to 1.1.3.

**Comment by SlimeDog** [comment]
Implemented in PluginVersions 1.2.0
Legacy formatting only
